packages/edit-post/src/components/sidebar/settings-header/index.js

```diff
@@ -11,11 +12,12 @@ import SidebarHeader from '../sidebar-header';
 
 const SettingsHeader = ( { openDocumentSettings, openBlockSettings, sidebarName } ) => {
 	const blockLabel = __( 'Block' );
+	// translators: ARIA label for the Document sidebar tab, not selected.
+	const documentLabel = applyFilters( 'editor.sidebar.settingsHeader.documentLabel', __( 'Document' ) );
```

Comment:
Also, I did not see any existing pattern to document `applyFilters`.

E.g., in PHP, we document the `apply_filters` wherever they are introduced along with the parameters and return values.

Do we have such pattern to add documentation for JS filters/hooks?

[Block Filters](https://developer.wordpress.org/block-editor/developers/filters/block-filters/) page does not seem like a good place for these extra utility hooks, as they are not directly related to blocks.